src/main/webapp/user_profile_script.js

```diff
@@ -12,17 +12,67 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function deleteRow(row) {
-    if (confirm("Are you sure that you want to delete this already published task?")) {
-        var rowIndex = row.parentNode.parentNode.rowIndex;
-        document.getElementById("need-help").deleteRow(rowIndex);
+async function getInfo(keyString) {
```

Comment:
"getTaskInfo" would better describe what this function does.

Reply:
Sure. I will change the function name in the new commit.